packages/common/package.json

```diff
@@ -16,7 +16,7 @@
     "./components/*": "./dist/components/*",
     "./hooks/*": "./dist/hooks/*",
     "./polyfills/*": "./dist/polyfills/*",
-    "./utils/*": "./dist/utils/*"
+    "./utils": "./dist/utils/index.ts"
```

Comment:
great 👍 
can you add index files to hooks and components ? ( delete policies)

Reply:
This change is a left over after some attempts to fix incremental compilation. It's here because in general it's nice if each module (or sub module) has a flat list of exports. In this case it was easy to implement. However in other packages we heavily rely on exporting nested paths. Changing that requires extra effort and is not needed to make the code work in dev mode.

cc: @sjd78  @sgratch